common_issues.md

```diff
@@ -1,6 +1,5 @@
 ---
 layout: page
-title: Common issues
```

Comment:
What will be the title then?

Reply:
As discussed in discord, the title will be the first heading of the content. In this case, it's `# Common issues`. The same in the other files.